crates/sui-framework/packages/deepbook/sources/clob_v2.move

```diff
@@ -216,7 +216,7 @@ module deepbook::clob_v2 {
         open_orders: LinkedTable<u64, Order>,
     }
 
-    struct Pool<phantom BaseAsset, phantom QuoteAsset> has key {
+    struct Pool<phantom BaseAsset, phantom QuoteAsset> has key, store {
```

Comment:
This is not useful unless we also expose an API that returns a `Pool`, right?

Reply:
Totally correct, just added new helper functions that expose the pool.